### ISSUE TITLE
Error using liballegro_ttf.so in Ubuntu 22.04.

### ISSUE THREAD

**Issue by paparoxo360** [liballeg/allegro5#1382]
I can't use ttf library on my program, I installed Allegro in Ubuntu 22.04, using allegro ppa, I used the command "sudo apt-get install liballegro*5.2 liballegro5-dev", everything is working great, but ttf is not working, I linked all the libraries in Codeblocks (except librallegro_ttf.so, it doesn't show in /usr/lib/x86_64-linux-gnu), I noted that all Allegro libraries(.so), are symlinks, so I created a symlink, therefore creating a librallegro_ttf.so file, but the error is still happening when I'm including <allegro5/allegro_ttf.h> in my program, if someone could help me? Thanks.

**Comment by SiegeLord** [comment]
Did you install `liballegro-ttf5-dev`?

**Comment by paparoxo360** [comment]
> Did you install `liballegro-ttf5-dev`?

I tried, but when I install it, it gets a dependency error, says it depends on libfreetype6-dev, then says that it depends on libfreetype6, but libfreetype6 is already installed, I think it needs an older version of the library, but I don't know what to do.

**Comment by SiegeLord** [comment]
Yeah, I'm not sure what's wrong. I just tried this in my installation of Ubuntu 22.04 and everything appeared to work okay. I called `sudo apt install liballegro-ttf5-dev` and it installed without error. I also compiled a test program, and that worked fine as well... I can only suggest removing and re-installing the Allegro packages.

**Comment by paparoxo360** [comment]
> Yeah, I'm not sure what's wrong. I just tried this in my installation of Ubuntu 22.04 and everything appeared to work okay. I called `sudo apt install liballegro-ttf5-dev` and it installed without error. I also compiled a test program, and that worked fine as well... I can only suggest removing and re-installing the Allegro packages.

I reinstalled it, and nothing yet, I can use every Allegro's library, the only one that I cannot use is ttf.h. But thanks for the help, I'll see what I can do to solve this problem, thanks again.

**Comment by rmbeer** [comment]
maybe is 'liballegro_ttf.so' and not 'librallegro_ttf.so'?
check (and show here) the list packages for "liballegro*5.2 liballegro5-dev" and all the files of each packages (finding any related for ttf libraries), and try check the correct name of files, and use 'ldd' command for each one for check the dependences.
